english/33hours/explore-outside/explore-outside-east/explore-east.md

```diff
@@ -1 +1,5 @@
-Suddenly a car shows up and it's heading right towards you.
\ No newline at end of file
+Suddenly, a car shows up and it's heading right towards you.
+
+[Car stops](car-stops/car-stops.md)
+
+[Car passes by](car-passes-by/car/passes-by.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.